WebRoot/js/zimbraMail/briefcase/model/ZmBriefcase.js

```diff
@@ -83,6 +83,7 @@ function() {
 ZmBriefcase.prototype.getIcon = 
 function() {
 	if (this.nId == ZmOrganizer.ID_ROOT)	{ return null; }
+	if (this.nId == ZmOrganizer.ID_FILE_SHARED_WITH_ME)	{ return "SharedContact"; }
```

Comment:
SharedContact ???

Reply:
Expected shareIcon is already present in the repo with "SharedContact" name.